python/GafferSceneUI/LightEditor.py

```diff
@@ -309,10 +309,17 @@ def __buttonDoubleClick( self, pathListing, event ) :
 
 	def __keyPress( self, pathListing, event ) :
 
-		if event.key == "Return" or event.key == "Enter" :
-			self.__editSelectedCells( pathListing )
+		if event.modifiers == event.Modifiers.None_ :
 
-			return True
+			if event.key == "Return" or event.key == "Enter" :
+				self.__editSelectedCells( pathListing )
+
+				return True
+
+			if event.key == "D" :
+				self.__removeEdits( pathListing )
```

Comment:
Would be good to rename `__removeEdits` to `__disableEdits` so we're using the same terminology throughout.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/4948/commits/1e84f34dc950b00a745d9790dc2f39fd6df6ac31 and https://github.com/GafferHQ/gaffer/pull/4948/commits/3494800095112a1673280dcfe1149485b5a5177c